src/main/java/com/bmc/truesight/saas/jira/JiraScriptApp.java

```diff
@@ -210,19 +212,28 @@ private static void readAndIngest(Template template) {
                 int recordsCount = jiraResponse.getValidEventList().size() + jiraResponse.getInvalidEventIdsList().size();
                 totalRecordsRead += recordsCount;
                 validRecords += jiraResponse.getValidEventList().size();
+                if (jiraResponse.getTotalCountAvailable() != availableRecordsSize) {
+                    log.info("Total available tickets count is changed from {} to {}", availableRecordsSize, jiraResponse.getTotalCountAvailable());
```

Comment:
Total available tickets matching the filter criteria has changed from {} to {}

^^^ may be re-articulate like this?